core/templates/pages/topics-and-skills-dashboard-page/modals/unassign-skill-from-topics-modal.component.html

```diff
@@ -4,14 +4,26 @@ <h3>
   </h3>
 </div>
 <div class="modal-body">
-  <span>This skill is assigned to these following topics, Tick the topics that you want this skill to be unassigned from.</span>
+  <div><em>The topics with this skill as their only diagnostic test skill are not eligible for the unassignment.</em></div><br>
```

Comment:
I don't really understand this -- we should just show those in the list and disable the checkbox with a reason, right?

Otherwise the user won't really know what to make of this. They'd see a list of topics and checkboxes but wouldn't know what to do to fix the issues that prevent the skill from being removed.

Reply:
Updated, based on the discussion in the meeting.